gui/main.qml

```diff
@@ -66,7 +66,7 @@ Item
         ScopeView
         {
             id: flowView
-            name: "Flow [mL]"
+            name: "Flow [mL/min]"
```

Comment:
Not to be changed here, fwiw I think this is probably the wrong unit.  TV is on the order if 1L per breath, and you do roughly a dozen breaths/min, so L/min seems like the right unit.  @SarahAdelaide may have an opinion, having gone through this constants exercise?

Reply:
Ack - I'm just using the unit specified in the proto.